packages/samples/specs/data-plane/language/authoring/authoring.tsp

```diff
@@ -80,15 +80,17 @@ model Project {
 }
 
 @doc("Represents the project kind.")
-enum ProjectKind {
+union ProjectKind {
+  string,
```

Comment:
nit, @doc?

Reply:
Not sure there is much value of describing this one. I don't think it is required by the required-documentation linting rule